bin/spice/cmd/search.go

```diff
@@ -58,6 +59,21 @@ spice search --cloud
 
 		rtcontext.RequireModelsFlavor(cmd)
 
+		datasets, err := api.GetDatasetsWithStatus(rtcontext)
+		if err != nil {
+			slog.Error("could not list datasets", "error", err)
+		}
+
+		for _, dataset := range datasets {
+			if dataset.Status != api.Ready.String() && dataset.Status != api.Refreshing.String() {
+				// warn only if vector_search is supported by the dataset
+				prop_val, _ := dataset.GetPropertyValue("vector_search")
+				if prop_val == "supported" {
+					slog.Warn(fmt.Sprintf("Dataset %s is not ready (%s) at the moment and will be excluded from the search if it is not ready when the search is performed.", dataset.Name, dataset.Status))
```

Comment:
```suggestion
					slog.Warn(fmt.Sprintf("Dataset %s is not ready (%s) and will be excluded from the search.", dataset.Name, dataset.Status))
```

Reply:
@phillipleblanc - this check is performed only once when the tool is started. As we don't collect datasets to pass to the search API now, there could be cases where a dataset is NOT ready at the start of the search but becomes ready and is included later when we work with the app

Phillip - could you please confirm the message should be updated.

Reply:
I don't think the addition words add anything to the message that the dataset is not ready? I would expect that when it becomes ready, it will be included in the search - but as of the moment the message is sent, it wouldn't be included, correct?

Reply:
>I would expect that when it becomes ready, it will be included in the search - but as of the moment the message is sent, it wouldn't be included, correct?

This is correct